cassiopeia/src/main/java/edu/asu/diging/gilesecosystem/cassiopeia/web/EditPropertiesController.java

```diff
@@ -23,79 +26,105 @@
 import edu.asu.diging.gilesecosystem.cassiopeia.web.validators.SystemConfigValidator;
 import edu.asu.diging.gilesecosystem.septemberutil.properties.MessageType;
 import edu.asu.diging.gilesecosystem.septemberutil.service.ISystemMessageHandler;
+import edu.asu.diging.gilesecosystem.septemberutil.service.impl.SystemMessageHandler;
 import edu.asu.diging.gilesecosystem.util.exceptions.PropertiesStorageException;
 import edu.asu.diging.gilesecosystem.util.properties.IPropertiesManager;
 
 @Controller
 public class EditPropertiesController {
-    
+
     @Autowired
     private IPropertiesManager propertyManager;
 
     @Autowired
     private ISystemMessageHandler messageHandler;
+    
+    @Autowired
+    private SystemMessageHandler msgHandler;
 
-    Map<String, String> ocrTypeMap = new HashMap<>();
+    private Map<String, String> ocrTypeMap = new HashMap<>();
+    private Map<String, String> langTypeMap = new HashMap<>();
+    private String defaultLang = new String("");
 
     @InitBinder
-    protected void initBinder(HttpServletRequest request, ServletRequestDataBinder binder, WebDataBinder validateBinder) {
+    protected void initBinder(HttpServletRequest request, ServletRequestDataBinder binder,
+            WebDataBinder validateBinder) {
         validateBinder.addValidators(new SystemConfigValidator());
         ocrTypeMap.put(Properties.OCR_PLAINTEXT, propertyManager.getProperty(Properties.OCR_PLAINTEXT));
         ocrTypeMap.put(Properties.OCR_HOCR, propertyManager.getProperty(Properties.OCR_HOCR));
+
     }
 
     @RequestMapping(value = "/admin/system/config", method = RequestMethod.GET)
     public String getConfigPage(Model model) {
         SystemConfigPage page = new SystemConfigPage();
-        
+
         page.setGilesAccessToken(propertyManager.getProperty(Properties.GILES_ACCESS_TOKEN));
         page.setBaseUrl(propertyManager.getProperty(Properties.BASE_URL));
 
-        if(propertyManager.getProperty(Properties.TESSERACT_CREATE_HOCR).equalsIgnoreCase("true")) {
+        TesseractOCRParser tessPars = new TesseractOCRParser(true,msgHandler);
+        List<String> langs = new ArrayList<String>();
+        langs.addAll(tessPars.getTessLangs(propertyManager)); 
+        
+        for (int i = 1; i < langs.size(); i++) {
+            langTypeMap.put(langs.get(i), langs.get(i));
+        }
+        tessPars = null;
+        if (langTypeMap.containsKey(Properties.ENGLISH)) {
+            defaultLang = langTypeMap.get(Properties.ENGLISH);
+        }
+        if (propertyManager.getProperty(Properties.TESSERACT_CREATE_HOCR).equalsIgnoreCase("true")) {
             page.setOCRType(Properties.OCR_HOCR);
         } else {
             page.setOCRType(Properties.OCR_PLAINTEXT);
         }
-
+        page.setLanguageType(defaultLang);
```

Comment:
let's move that to line 76 to keep it together with the rest of the default language code